packaging/tests/tas-restapi/src/test/java/org/alfresco/rest/rules/RulesTestsUtils.java

```diff
@@ -95,8 +95,8 @@ public static RestRuleModel createRuleModel(String name, List<RestActionBodyExec
     public static RestActionBodyExecTemplateModel createDefaultActionModel()
     {
         RestActionBodyExecTemplateModel restActionModel = new RestActionBodyExecTemplateModel();
-        restActionModel.setActionDefinitionId("add-features");
-        restActionModel.setParams(Map.of("aspect-name", "cm:audio"));
+        restActionModel.setActionDefinitionId("set-property-value");
+        restActionModel.setParams(Map.of("aspect-name", "cm:audio", "actionContext", "rule"));
```

Comment:
Entry of `"actionContext" -> "rule"` should not be in the action parameters when creating new action. 
This entry is added automatically while validating rule actions when creating/updating rule (see [ACS-3335](https://alfresco.atlassian.net/browse/ACS-3335) and [ACS-3336](https://alfresco.atlassian.net/browse/ACS-3336))
If you are using this method to prepare input model for newly created or updated rule with new action - you should not  add `"actionContext" -> "rule"` entry in action parameters.
There code with a method adding that entry to each action (`RulesTestsUtils::addActionContextParams`) should already be merged. Mentioned method can be used to enhance rule action when creating expected rule object.

Reply:
I saw your changes are already merged and modified the actions, thanks 🙏